ConsoleView.h

```diff
@@ -17,32 +18,31 @@ class ConsoleView
 {
 
 public:
-    ConsoleView(){};
-  
-    
-    
-    
     // PrintClass Name will be used to Print the names of the class being passed by the user
     //@ Para: UMLClass object 
-    void PrintClassName(UMLClass X){}
+    void Print_Class_Name(UMLClass X);
```

Comment:
You changed the cpp file to have lowercase method names with underscores (yay!) but now the header file doesn't match!

Reply:
lol all good